.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
this wont have any effect on contributors who don't manually (and explicitly) allow this to be used. I recommend everybody to select "Select TypeScript's Version..." from the command palette and choose the workspace version. It can save you from seeing a class of mismatched results between the VS Code and `tsc`.

If you open multiple projects in a single workspace... I'm not sure how that works though. I don't think VS Code is able to use different TS versions between such projects (like when u open both backend and reaply-cli in a single VS Code instance)